lms/djangoapps/commerce/views.py

```diff
@@ -54,17 +54,16 @@ def _is_data_valid(self, request):
 
         return True, course_key, None
 
-    def _get_jwt(self, user):
+    def _get_jwt(self, user, ecommerce_api_signing_key):
```

Comment:
The existence of the signing key [is verified](https://github.com/edx/edx-platform/pull/7375/files#diff-3e84d5041cfb763b355bac2d2d063ae9R85) before this helper is ever called, so I don't think this change is necessary.